packages/loopring_v3/contracts/impl/ExchangeV3.sol

```diff
@@ -218,21 +218,28 @@ contract ExchangeV3 is IExchangeV3
         override
         nonReentrant
         payable
-        onlyAgentFor(owner)
         returns (
             uint24 accountID,
             bool   isAccountNew,
             bool   isAccountUpdated
         )
     {
+        try state.getAccountID(owner) {
+            // Account exist, perform update
+            require(isAgent(owner, msg.sender), "UNAUTHORIZED");
+        } catch {
+             // Account not exist perform creation
+            require(owner == msg.sender || pubKeyX == 0 && pubKeyY == 0, "PUBKEY_MUST_BE_ZERO");
```

Comment:
Use `pubKeyX == 1` here for example. `pubKeyX == 0` is detected in the circuits as an unregistered account and nothing can be transferred to an account like that.